scripts/docker/bokeh_docker_from_wheel.sh

```diff
@@ -9,10 +9,11 @@ pip install dist/bokeh*.whl
 bash scripts/ci/install_node_modules.sh
 
 python -m bokeh info
+bokeh sampledata
 
 # Bokeh Python tests.
-bokeh sampledata && \
-pytest tests/unit -k "not firefox" && \
+pytest tests/test_defaults.py
+pytest tests/unit -k "not firefox"
 pytest tests/integration
```

Comment:
All tests need to run even if one group fails. We (or at least I) don't want to run this multiple times to have a complete picture of how many failures there actually are. Or maybe this is somehow setup like this, just it's not clear to me from this script?

Reply:
This is a work in progress and you shouldn't be spending your time reviewing it when you have many other more important things to work on.

When this is eventually merged, test failures that occur in the docker container will not be cause to stop PRs from being merged. There will need to be an extended period of using the docker container in CI to find and eliminate all of the problems before we can consider it robust.